### PR TITLE
enhance: refine parameter relationship for hybridsearch_group_by(#35096)

### PR DESCRIPTION
**Pull request by MrPresent-Han**
related: #35096

### CODE REVIEW

File: internal/proxy/search_util.go [hunk 1]
```diff
@@ -141,51 +165,16 @@ func parseSearchInfo(searchParamsPair []*commonpb.KeyValuePair, schema *schemapb
...
-			fmt.Sprintf("input group size:%d exceeds configured max group size:%d", groupSize, Params.QuotaConfig.MaxGroupSize.GetAsInt64()))
-	}
-
+	var groupByFieldId, groupSize int64
 	var groupStrictSize bool
-	groupStrictSizeStr, err := funcutil.GetAttrByKeyFromRepeatedKV(GroupStrictSize, searchParamsPair)
-	if err != nil {
-		groupStrictSize = false
+	if isAdvanced {
+		groupByFieldId, groupSize, groupStrictSize = rankParams.groupByFieldId, rankParams.groupSize, rankParams.groupStrictSize
```
Comment (czs007) [inline internal/proxy/search_util.go:171]: rankParams.GetXXX instead of rankParams.groupByFieldId

Comment (MrPresent-Han) [reply internal/proxy/search_util.go]: changed

**Comment by MrPresent-Han** [comment]
rerun ut

**Comment by MrPresent-Han** [comment]
rerun ut

**Pull request merged**